### PR TITLE
Enable json logging feature for cloud native

### PR DESCRIPTION
**Pull request by alfsch**
To integrate better with cloud native environments ELK stack json logging is needed.

- enable quarkus json logging feature
- enable also mandatory quarkus jsonp feature
- enhance application.properties to log without json in devmode

### CODE REVIEW

File: app/src/main/resources/application.properties [hunk 1]
```diff
@@ -12,4 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-quarkus.log.level=${KAFKAPROXY_LOG_LEVEL:INFO}
\ No newline at end of file
+quarkus.log.level=${KAFKAPROXY_LOG_LEVEL:INFO}
+%dev.quarkus.log.console.json=false
```
Comment (dajudge) [inline app/src/main/resources/application.properties:16]: How about this?
`quarkus.log.console.json=${KAFKAPROXY_ENABLE_JSON_LOGGING:false}`

Comment (alfsch) [reply app/src/main/resources/application.properties:16]: Fine, let's do it so.

Comment (dajudge) [reply app/src/main/resources/application.properties:16]: Nice :-) Can you adjust the PR accordingly?

Comment (alfsch) [reply app/src/main/resources/application.properties:16]: I'll do and also adjust the Readme with documentation for the new Env var.

**Pull request merged**